config/karma.conf.js

```diff
@@ -87,7 +87,7 @@ module.exports = function(config) {
      * Continuous Integration mode
      * if true, Karma captures browsers, runs the tests and exits
      */
-    singleRun: true
+    singleRun: false
```

Comment:
I updated the `package.json` setting the `--single-run` option when execute `npm run test`. I have to do this because setting `singleRun: true` in the karma config don't allows to use the `--auto-watch` option.